client/components/map.jsx

```diff
@@ -44,18 +44,30 @@ class Map extends Component {
     })
 
 }
-
-  componentDidMount(){
-  
+  loadMapScript() {
     const googleMapScript = document.createElement('script');
     googleMapScript.src = 
       `https://maps.googleapis.com/maps/api/js?key=${GOOGLE_MAP_API_KEY}`;
     
     window.document.body.appendChild(googleMapScript);
-    googleMapScript.addEventListener('load', () => {
+
+  }
+
+  componentDidMount(){
+    if(!window.google) {
```

Comment:
Oooh fancy! window.google?? What does that do

Reply:
that checks if the script is there